doc/source/drivers/raster/vrt.rst

```diff
@@ -1669,6 +1674,9 @@ The effect of the ``a_ullr`` option (added in GDAL 3.7) is to assign (override)
 bounds of the source in the same way as (:ref:`gdal_translate`). The value consists of four numeric
 values separated by commas, in the order 'xmin,ymax,xmax,ymin' (upper left x,y, lower right x,y).
 
+The effect of the ``ovr``` option (added in GDAL 3.7) is to specify which overview 
+level of source file must be used, or control any automatic choice as with (:ref:`gdal_translate`).
+
```

Comment:
AFIACS, the automatic choices won't work there as there's no size constraints
```suggestion
The effect of the ``ovr``` option (added in GDAL 3.7) is to specify which overview 
level of source file must be used, with the first overview level being 0 (:ref:`gdal_translate`).

```

Reply:
I see, yes